### ISSUE TITLE
Kiosk: site notes in director's view

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#888]
Absolutely immediate first thing I miss in turning to the Director's View. So I make a note of it, but I am a bit skeptical - of all the things the Director shouldn't need a daily view of it is the section that is designed for the Director to make notes in...

**Comment by urapadmin** [comment]
yes, that is the reason why I did not include the site notes in either the narrative or the file widget. I am just not sure if all projects would follow your lead and have site notes used by the director only.

**Comment by luizaogs** [comment]
Here we use site notes as notes for the tomb as a whole, so having it visible in the director's view would indeed be useful because I am not the only one making them. This is not something I need an update for during the season, Lutz, but just letting you know of a project that would benefit from having this in general.

**Comment by luizaogs** [comment]
Lutz, since you're currently working on DV -- I wanted to flag this as something that would be helpful for the upcoming MAD-P season for the reason stated above. If we get security clearance, the plan would be to go in January.

**Comment by urapadmin** [comment]
wait - did I get that wrong and it is MAD-P instead of MKAP that is scheduled for the end of the year? That you have to go back to MAD-P without image annotation actually hurts 😯 

Otherwise, yes, goo to know that you'll need this. It also reminds me of #2377

**Comment by luizaogs** [comment]
It’s both. But yes, #2377 would also be helpful!

**Comment by urapadmin** [comment]
delighted to hear that you guys are not giving up ... ⛏️

**Comment by luizaogs** [comment]
Well. Not yet…

**Comment by urapadmin** [comment]
the DV is actually designed to show all narratives and descriptions together in the Narratives widget. So from now on this simply includes site notes as well.

And the files widget includes site note photos.

**Comment by urapadmin** [comment]
### Kiosk 1.7.20
- site notes appear now in the narrative widget if altered / created
- site note photos appear in the file widget if altered / created
- site maps appear in the file widget if the site record got changed (which includes changing the image, but I cannot distinguish between changing the image and the other data of the site record)

**Comment by luizaogs** [comment]
Works. Thank you.
